### ISSUE TITLE
Callback URL error message from Barion

### ISSUE THREAD

**Issue by marceldarvas** [szelpe/woocommerce-barion#19]
experience a callback issue.
In the email it shows, the cURL sent the following callback url:

Megadott callback URL: https%3A%2F%2F_URL MASKED FROM THIS FIELD_%2Fwc-api%2FWC_Gateway_Barion%2F%3FpaymentId%3D_PAYMENT ID MASKED FROM HERE_
Kapott HTTP státusz kód: 500 (InternalServerError)

I'm using the plugin version **2.1.0** with the following configuration.
```
WC Version: 3.3.4
Log Directory Writable: ✔
WP Version: 4.9.4
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
Language: hu_HU

### Server Environment ###

Server Info: Apache/2.4.28 (Unix) OpenSSL/1.0.2g
PHP Version: 7.1.15
PHP Post Max Size: 1 GB
PHP Time Limit: 30
PHP Max Input Vars: 5000
cURL Version: 7.47.0
OpenSSL/1.0.2g

SUHOSIN Installed: –
MySQL Version: 5.7.21
Max Upload Size: 512 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔
```

I don't have any callback URL configured in the Barion Shop panel.
For "Elfogadóhely visszatérési URL-je:" I have my URL+/penztar/success/ saved, as I was unable to set it up with shop root URL.

I don't see any PHP errors in my logs, but could it be the 7.1 version?

**Comment by szelpe** [comment]
Hi Marcel,

Please check if there are any error logged for WooCommerce (you can find the logs under WooCommerce > Status > Logs, or WooCommerce > Állapot > Naplók in Hungarian)

**Comment by szelpe** [comment]
Also, make sure that the callback URL `/wc-api/WC_Gateway_Barion?paymentId=*` is accessible for the Barion server. (potential issue: your site is protected with Basic Authentication and Barion can't call the URL)

**Comment by marceldarvas** [comment]
Hi Peter,

Thank you for your quick answer! Unfortunately I wasn't able to find any log files since we've been getting an error for this. I do use CloudFlare, so maybe that could be the issue.
Now I added a rule for bypassing Security on that URL and I'll just wait till the next time a customer uses Barion as a Payment Method.
I see a minor update for the Barion WooCommerce plugin was also released, let's see if these resolve the issue.
